pkg/kubefed2/federate/disable.go

```diff
@@ -119,9 +119,14 @@ func (j *disableType) Run(cmdOut io.Writer, config util.FedConfig) error {
 		return errors.Wrap(err, "Failed to get host cluster config")
 	}
 
+	apiResource, err := LookupAPIResource(hostConfig, j.targetName, "")
```

Comment:
If the target name is qualified (i.e. includes a `.`), I think it can be assumed that the user is providing the group-qualified name likely to correspond to the ftc name.

Reply:
make sense. Resolved